pkg/cmd/dictionary/entries/browse/browse.go

```diff
@@ -0,0 +1,238 @@
+			$ algolia dictionary entries browse stopword
+
+			# Retrieve all entries from the "stopword" and "plural" dictionnaries
+			$ algolia dictionary entries browse stopword plural
+
+			# Retrieve all entries from all dictionnaries
+			$ algolia dictionary entries browse --all
+
+			# Retrieve all entries from the "stopword" dictionnaries (including default stopwords)
+			$ algolia dictionary entries browse stopword --showDefaultStopwords
+		`),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if opts.All && len(args) > 0 || !opts.All && len(args) == 0 {
+				return cmdutil.FlagErrorf("Either specify dictionaries' names or use --all to browse all dictionaries")
+			}
+
+			if opts.All {
+				opts.Dictionnaries = []search.DictionaryName{search.Stopwords, search.Plurals, search.Compounds}
+			} else {
+				opts.Dictionnaries = make([]search.DictionaryName, len(args))
+				for i, dictionnary := range args {
+					opts.Dictionnaries[i] = search.DictionaryName(dictionnary)
+				}
+			}
+
+			return runBrowseCmd(opts)
+		},
+	}
+
+	cmd.Flags().BoolVarP(&opts.All, "all", "a", false, "browse all dictionnaries")
+	cmd.Flags().BoolVar(&opts.ShowDefaultStopwords, "showDefaultStopwords", false, "browse dictionaries and include default stopwords")
+
+	opts.PrintFlags.AddFlags(cmd)
+
+	return cmd
+}
+
+// runBrowseCmd executes the browse command
+func runBrowseCmd(opts *BrowseOptions) error {
+	cs := opts.IO.ColorScheme()
+	client, err := opts.SearchClient()
+	if err != nil {
+		return err
+	}
+
+	dictionaries := opts.Dictionnaries
+	dictionariesNames := make([]string, len(dictionaries))
+
+	dictionariesCustomEntriesNb := make([]int, len(dictionaries))
+	dictionariesAllEntriesNb := make([]int, len(dictionaries))
+
+	dictionariesCustomEntries := make([][]DictionaryEntry, len(dictionaries))
+	dictionariesAllEntries := make([][]DictionaryEntry, len(dictionaries))
+
+	for i, dictionnary := range dictionaries {
+		// get all entries, custom entries, and number of both entries
+		allEntries, customEntries, nbAllEntries, nbCustomEntries, err := dictionaryEntriesData(client, dictionnary)
+		if err != nil {
+			return err
+		}
+		dictionariesCustomEntriesNb[i] = nbCustomEntries
+		dictionariesAllEntriesNb[i] = nbAllEntries
+
+		dictionariesNames[i] = string(dictionnary)
+
+		dictionariesCustomEntries[i] = customEntries
+		dictionariesAllEntries[i] = allEntries
+	}
+
+	totalCustomEntries := 0
+	totalAllEntries := 0
+
+	p, err := opts.PrintFlags.ToPrinter()
+	if err != nil {
+		return err
+	}
+
+	// incase the `--showDefaultDictionary` flag is set
+	if opts.ShowDefaultStopwords {
+		for _, nb := range dictionariesAllEntriesNb {
+			totalAllEntries += nb
+		}
+		if totalAllEntries == 0 {
+			fmt.Fprintf(opts.IO.Out, "%s No entries in %s dictionary.\n", cs.WarningIcon(), utils.SliceToReadableString(dictionariesNames))
+			return nil
+		}
+		for _, dictHits := range dictionariesAllEntries {
+			// print dictionary entries
+			p.Print(opts.IO, dictHits)
```

Comment:
Right now, we want to only return one single object per line (nd-json), I don't think this is the case here.